cluster/gce/gci/configure-helper.sh

```diff
@@ -752,10 +752,10 @@ function start-kube-apiserver {
   fi
   local -r src_dir="${KUBE_HOME}/kube-manifests/kubernetes/gci-trusty"
 
-  if [[ -n "${KUBE_USER:-}" ]]; then
+  if [[ -n "${KUBE_USER}" ]]; then
```

Comment:
> No, ${KUBE_USER:-} doesn't set KUBE_USER. Use ${KUBE_USER:=admin} if you want to initialize it.

Ah. I assumed it also sets it. I restored it as before. But I think with `set -o nounset` at the top of this file, even `${KUBE_USER:-}` will cause this code to fail if KUBE_USER is unset. I also see KUBE_USER used with the not-null check above.